### PR TITLE
Yes

### PR DESCRIPTION
**Pull request by 22nadsup**
SPACING

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Hopper.java [hunk 1]
```diff
@@ -18,19 +18,19 @@
     private boolean mIsBrakeMode;
 
     // Logging
-    private final Logger mLogger = LoggerFactory.getLogger( Hopper.class );
+    private final Logger mLogger = LoggerFactory.getLogger ( Hopper.class );
```
Comment (DianaBellefeuille) [inline src/main/java/frc/robot/subsystems/Hopper.java]: undo this, we only want spaces in the parenthesis, not between the parenthesis and function.

File: src/main/java/frc/robot/subsystems/Hopper.java [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
 
 
     public void SetBrakeMode ( boolean wantsBrakeMode ) {
-        if (wantsBrakeMode && !mIsBrakeMode) {
+        if ( wantsBrakeMode && !mIsBrakeMode ) {
             mIsBrakeMode = wantsBrakeMode;
-            mHopperMotor.setIdleMode(IdleMode.kBrake);
-      mLogger.info("Neutral mode set to: [Brake]");
+            mHopperMotor.setIdleMode( IdleMode.kBrake );
+                mLogger.info( "Neutral mode set to: [Brake]" );
```
Comment (DianaBellefeuille) [inline src/main/java/frc/robot/subsystems/Hopper.java]: Why is this tabbed over out of line?

File: src/main/java/frc/robot/subsystems/Hopper.java [hunk 3]
```diff
@@ -18,19 +18,19 @@
...
 
-    } else if (!wantsBrakeMode && mIsBrakeMode) {
-      mIsBrakeMode = wantsBrakeMode;
-      mHopperMotor.setIdleMode(IdleMode.kCoast);
-            mLogger.info( "Neutral mode set to: [Coast]" );
+        } else if (!wantsBrakeMode && mIsBrakeMode) {
+            mIsBrakeMode = wantsBrakeMode;
+            mHopperMotor.setIdleMode( IdleMode.kCoast );
+                mLogger.info( "Neutral mode set to: [Coast]" );
```
Comment (DianaBellefeuille) [inline src/main/java/frc/robot/subsystems/Hopper.java]:
```suggestion
           mLogger.info( "Neutral mode set to: [Coast]" );
```

**Review by DianaBellefeuille — COMMENTED** [review]
Please fix in line issues

**Review by DianaBellefeuille — APPROVED** [review]
good space fellow introvert

**Review by ghost — APPROVED** [review]
Good practice.  Now make robot move good.

**Pull request merged**